### DISCUSSION TITLE
Information

### DISCUSSION THREAD

**Discussion by jesutobi** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#23] [Outreachy 2022] [answered]
I found it difficult to make a pull request initially, but i got through.
so in case there is anyone finding it difficult also, here are a few steps that would help...I hope it helps

- fork the Repo, this would create a copy of the repository in your list of repositories
![fork](https://user-images.githubusercontent.com/43034407/194888636-bf108b01-9968-455f-b9e1-664ccd0b5da6.png)

- Now clone the forked repository:you can find it here
![repo](https://user-images.githubusercontent.com/43034407/194889197-5212d916-0e4a-472c-8b9f-012ab27d259d.png)

- to clone:click on the green button that's called "code" in your forked repository, then copy the URL
![clone](https://user-images.githubusercontent.com/43034407/194889823-6adf1263-3b00-4189-88ff-85906f2dc8b1.png)

- now open your vscode, open terminal, then type ''git clone copied URL''  (don't add the quote).
- now also create a new branch, still on that terminal, with "git branch branchname"
- to move to the new branch you just created,type "git checkout branchname"
- after contributing,open the terminal again, then type

-  "git add . "
- wait for it to load
- git commit -m'short description of what you need'....(dont remove the quote )
- - wait for it to load
- git push origin branchname
- - wait for it to load

Then go to the Repo you forked from on git hub......immediately you open it you would see a button  that says "create pull request"
then you are good to go from there.

i really hope this helps someone thats finding it difficult

**Comment by BatoolMM** [comment, marked as answer]
Thank you @jesutobi, that's kind of you.
Would you like to submit a PR for `CONTRIBUTING.md` where you explain how they can make contributions in the website?

This is [an example of `CONTRIBUTING.md` file in other repo.](https://github.com/Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi/blob/main/CONTRIBUTING.md)

**Reply by jesutobi** [reply]
You are welcome @BatoolMM .
Goodmorning,
Yes, i would like that, would get to it right away, would also revert to you, incase i have questions.Thanks
